### ISSUE TITLE
Changes for easyer development

### ISSUE THREAD

**Issue by samuel-hxs** [FAForever/client#1061]
**I experienced..** 
Forked and cloned client.
Opend in pycharm. 
Created new run configuration.
Using Python 3.6.

**how to reproduce** if explanation is needed to reproduce the problem, add it here
Adding the client as modul to the run configuration creates missing utils problem.

**I expected...** write here what you think the best behaviour should be
Run the module without changing the working directory into src.

**Where the bug is / What should be done** if you have a suggestion for fixing the issue, add it here.
Check imports to make it possible running src as module like it is proberly intendet by Python designers.
Rename src to something like faf_python_client.

**Comment by Wesmania** [comment]
Yeah, this is a TODO. Right now the client puts his own path into PYTHONPATH in order to import everything locally. AFAIK right now the convention is to use absolute imports everywhere, so what we'll need to do is replace all the relative imports with something like 'from fafclient import x'. We'll need to check how this change affects cx_freeze, too.
